Userland/DevTools/UserspaceEmulator/Emulator.cpp

```diff
@@ -51,7 +51,7 @@ Emulator::Emulator(String const& executable_path, Vector<StringView> const& argu
 {
     m_malloc_tracer = make<MallocTracer>(*this);
 
-    static constexpr FlatPtr userspace_range_base = 0x00800000;
+    static constexpr FlatPtr userspace_range_base = 0x10000;
```

Comment:
Should we expose the `USER_RANGE_BASE` constant to userspace?

Reply:
Probably, yes, I just wasn't sure what the best way to do that would be, any ideas?

Reply:
I guess we could do it with a header, `Kernel/API/MemoryLayout.h` or something? 

Alternatively we could expose it as a `sysfs`/`procfs` entry?

Reply:
Since it's just a constant right now I don't think there's a point to exposing it via sysfs/procfs, `Kernel/API/MemoryLayout.h` sounds good to me, added.